api/policies.kyverno.io/v1alpha1/imagevalidating_policy.go

```diff
@@ -403,6 +447,9 @@ type Key struct {
 	// SecretRef sets a reference to a secret with the key.
 	// +optional
 	SecretRef *corev1.SecretReference `json:"secretRef,omitempty"`
+	// CEL is a CEL expression that returns the Key.
+	// +optional
+	CEL string `json:"cel,omitempty"`
```

Comment:
Does `cel` overlap with `secretRef`? 

Can we remove `secretRef` if the secret can be fetched via CEL functions?